crates/bevy_internal/Cargo.toml

```diff
@@ -97,6 +97,8 @@ bevy_render = ["dep:bevy_render", "bevy_scene?/bevy_render"]
 # Enable assertions to check the validity of parameters passed to glam
 glam_assert = ["bevy_math/glam_assert"]
 
+default_font = ["bevy_text?/default_font"]
```

Comment:
@mockersf is that `?` a typo or a special syntax I don't know? 😊

Reply:
Not a typo! https://doc.rust-lang.org/cargo/reference/features.html#dependency-features

it means "enable `default_font` feature of optional crate `bevy_text` only if the crate if enabled from another path, otherwise just ignore it"

Reply:
Sweet, I didn't know about that feature, quite useful. Thanks!